hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoDeleteAllowanceHandlerTest.java

```diff
@@ -284,6 +328,58 @@ void considersPayerIfOwnerNotSpecified() {
         assertThat(writableNftStore.get(nftIdSl2).spenderId()).isNull();
     }
 
+    @Test
+    @DisplayName("check that fees are 1 for delete account allowance trx")
+    void testCalculateFeesReturnsCorrectFeeForDeleteAccountAllowance() {
+        final var feeCtx = mock(FeeContext.class);
+        final var feeCalcFact = mock(FeeCalculatorFactory.class);
+        final var feeCalc = mock(FeeCalculator.class);
+        final var txnBody = cryptoDeleteAllowanceTransaction(payerId);
+        given(feeCtx.feeCalculatorFactory()).willReturn(feeCalcFact);
+        given(feeCtx.body()).willReturn(txnBody);
+        given(feeCalcFact.feeCalculator(any())).willReturn(feeCalc);
+        given(feeCalc.addBytesPerTransaction(anyLong())).willReturn(feeCalc);
+        given(feeCalc.calculate()).willReturn(new Fees(1, 0, 0));
+
+        assertThat(subject.calculateFees(feeCtx)).isEqualTo(new Fees(1, 0, 0));
+    }
+
+    @Test
```

Comment:
suggested addition of 
`@DisplayName("calculate fees correctly considering bytes per transaction")`

Reply:
added, ty @thomas-swirlds-labs